File: Intent2/app/src/test/java/Modelo/Pedido.java
package Modelo;

import java.util.ArrayList;

public class Pedido {
    private ArrayList<Pizza> arrayPizzasPedidas;

    public Pedido(ArrayList<Pizza> arrayPizzasPedidas) {
        this.arrayPizzasPedidas = arrayPizzasPedidas;
    }

    public ArrayList<Pizza> getPizza() {
        return arrayPizzasPedidas;
    }

    public void setPizza(ArrayList<Pizza> arrayPizzasPedidas) {
        this.arrayPizzasPedidas = arrayPizzasPedidas;
    }
}

File: Pizzeria/app/src/main/java/Modelo/Pedido.kt
package Modelo

class Pedido {

    private var arrayPizzasPedidas: ArrayList<Pizza>? = null


    constructor(arrayPizzasPedidas: ArrayList<Pizza>?) {
        this.arrayPizzasPedidas = arrayPizzasPedidas
    }

    fun getPizza(): ArrayList<Pizza>? {
        return arrayPizzasPedidas
    }

    fun setPizza(arrayPizzasPedidas: ArrayList<Pizza>?) {
        this.arrayPizzasPedidas = arrayPizzasPedidas
    }

}